radixdlt/babylon-wallet-android
# Sargon transaction analysis and submit

Jira Tickets:
- [ABW-3863](https://radixdlt.atlassian.net/browse/ABW-3863)
- [ABW-3859](https://radixdlt.atlassian.net/browse/ABW-3859)

Depends on:
- https://github.com/radixdlt/sargon/pull/230
- https://github.com/radixdlt/sargon/pull/231
 
## Description
This PR uses SargonOS to get the Transaction Preview, submit the transaction, and poll the transaction status.


## How to test

Test any transaction and check that everything works as expected.


[ABW-3863]: https://radixdlt.atlassian.net/browse/ABW-3863?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[ABW-3859]: https://radixdlt.atlassian.net/browse/ABW-3859?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
